### ISSUE TITLE
14.0.0-rc.2 cannot be used in Angular app

### ISSUE THREAD

**Issue by Ben305** [graphql/graphql-js#1394]
I tried to use the prerelease version in an Angular 6 app, but the app could not be compiled:

<pre>
ERROR in ./node_modules/graphql/index.mjs
34:0-49 Can't reexport the named export 'graphql' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
34:0-49 Can't reexport the named export 'graphqlSync' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
36:0-47:50 Can't reexport the named export 'DEFAULT_DEPRECATION_REASON' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
36:0-47:50 Can't reexport the named export 'GraphQLBoolean' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
36:0-47:50 Can't reexport the named export 'GraphQLDeprecatedDirective' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
36:0-47:50 Can't reexport the named export 'GraphQLDirective' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
36:0-47:50 Can't reexport the named export 'GraphQLEnumType' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
36:0-47:50 Can't reexport the named export 'GraphQLFloat' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
36:0-47:50 Can't reexport the named export 'GraphQLID' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
36:0-47:50 Can't reexport the named export 'GraphQLIncludeDirective' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
36:0-47:50 Can't reexport the named export 'GraphQLInputObjectType' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
36:0-47:50 Can't reexport the named export 'GraphQLInt' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
36:0-47:50 Can't reexport the named export 'GraphQLInterfaceType' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
36:0-47:50 Can't reexport the named export 'GraphQLList' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
36:0-47:50 Can't reexport the named export 'GraphQLNonNull' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
36:0-47:50 Can't reexport the named export 'GraphQLObjectType' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
36:0-47:50 Can't reexport the named export 'GraphQLScalarType' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
36:0-47:50 Can't reexport the named export 'GraphQLSchema' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
36:0-47:50 Can't reexport the named export 'GraphQLSkipDirective' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
36:0-47:50 Can't reexport the named export 'GraphQLString' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
36:0-47:50 Can't reexport the named export 'GraphQLUnionType' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
36:0-47:50 Can't reexport the named export 'SchemaMetaFieldDef' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
36:0-47:50 Can't reexport the named export 'TypeKind' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
36:0-47:50 Can't reexport the named export 'TypeMetaFieldDef' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
36:0-47:50 Can't reexport the named export 'TypeNameMetaFieldDef' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
36:0-47:50 Can't reexport the named export '__Directive' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
36:0-47:50 Can't reexport the named export '__DirectiveLocation' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
36:0-47:50 Can't reexport the named export '__EnumValue' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
36:0-47:50 Can't reexport the named export '__Field' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
36:0-47:50 Can't reexport the named export '__InputValue' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
36:0-47:50 Can't reexport the named export '__Schema' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
36:0-47:50 Can't reexport the named export '__Type' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
36:0-47:50 Can't reexport the named export '__TypeKind' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
36:0-47:50 Can't reexport the named export 'assertAbstractType' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
36:0-47:50 Can't reexport the named export 'assertCompositeType' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
36:0-47:50 Can't reexport the named export 'assertEnumType' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
36:0-47:50 Can't reexport the named export 'assertInputObjectType' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
36:0-47:50 Can't reexport the named export 'assertInputType' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
36:0-47:50 Can't reexport the named export 'assertInterfaceType' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
36:0-47:50 Can't reexport the named export 'assertLeafType' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
36:0-47:50 Can't reexport the named export 'assertListType' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
36:0-47:50 Can't reexport the named export 'assertNamedType' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
36:0-47:50 Can't reexport the named export 'assertNonNullType' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
36:0-47:50 Can't reexport the named export 'assertNullableType' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
36:0-47:50 Can't reexport the named export 'assertObjectType' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
36:0-47:50 Can't reexport the named export 'assertOutputType' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
36:0-47:50 Can't reexport the named export 'assertScalarType' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
36:0-47:50 Can't reexport the named export 'assertType' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
36:0-47:50 Can't reexport the named export 'assertUnionType' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
36:0-47:50 Can't reexport the named export 'assertValidSchema' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
36:0-47:50 Can't reexport the named export 'assertWrappingType' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
36:0-47:50 Can't reexport the named export 'getNamedType' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
36:0-47:50 Can't reexport the named export 'getNullableType' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
36:0-47:50 Can't reexport the named export 'introspectionTypes' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
36:0-47:50 Can't reexport the named export 'isAbstractType' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
36:0-47:50 Can't reexport the named export 'isCompositeType' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
36:0-47:50 Can't reexport the named export 'isDirective' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
36:0-47:50 Can't reexport the named export 'isEnumType' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
36:0-47:50 Can't reexport the named export 'isInputObjectType' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
36:0-47:50 Can't reexport the named export 'isInputType' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
36:0-47:50 Can't reexport the named export 'isInterfaceType' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
36:0-47:50 Can't reexport the named export 'isIntrospectionType' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
36:0-47:50 Can't reexport the named export 'isLeafType' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
36:0-47:50 Can't reexport the named export 'isListType' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
36:0-47:50 Can't reexport the named export 'isNamedType' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
36:0-47:50 Can't reexport the named export 'isNonNullType' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
36:0-47:50 Can't reexport the named export 'isNullableType' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
36:0-47:50 Can't reexport the named export 'isObjectType' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
36:0-47:50 Can't reexport the named export 'isOutputType' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
36:0-47:50 Can't reexport the named export 'isScalarType' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
36:0-47:50 Can't reexport the named export 'isSchema' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
36:0-47:50 Can't reexport the named export 'isSpecifiedDirective' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
36:0-47:50 Can't reexport the named export 'isSpecifiedScalarType' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
36:0-47:50 Can't reexport the named export 'isType' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
36:0-47:50 Can't reexport the named export 'isUnionType' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
36:0-47:50 Can't reexport the named export 'isWrappingType' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
36:0-47:50 Can't reexport the named export 'specifiedDirectives' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
36:0-47:50 Can't reexport the named export 'specifiedScalarTypes' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
36:0-47:50 Can't reexport the named export 'validateSchema' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
49:0-52:117 Can't reexport the named export 'BREAK' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
49:0-52:117 Can't reexport the named export 'DirectiveLocation' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
49:0-52:117 Can't reexport the named export 'Kind' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
49:0-52:117 Can't reexport the named export 'Source' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
49:0-52:117 Can't reexport the named export 'TokenKind' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
49:0-52:117 Can't reexport the named export 'getLocation' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
49:0-52:117 Can't reexport the named export 'getVisitFn' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
49:0-52:117 Can't reexport the named export 'parse' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
49:0-52:117 Can't reexport the named export 'parseType' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
49:0-52:117 Can't reexport the named export 'parseValue' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
49:0-52:117 Can't reexport the named export 'print' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
49:0-52:117 Can't reexport the named export 'visit' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
49:0-52:117 Can't reexport the named export 'visitInParallel' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
49:0-52:117 Can't reexport the named export 'visitWithTypeInfo' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
54:0-101 Can't reexport the named export 'defaultFieldResolver' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
54:0-101 Can't reexport the named export 'execute' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
54:0-101 Can't reexport the named export 'getDirectiveValues' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
54:0-101 Can't reexport the named export 'responsePathAsArray' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
55:0-68 Can't reexport the named export 'createSourceEventStream' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
55:0-68 Can't reexport the named export 'subscribe' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
57:0-59:675 Can't reexport the named export 'FieldsOnCorrectTypeRule' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
57:0-59:675 Can't reexport the named export 'FragmentsOnCompositeTypesRule' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
57:0-59:675 Can't reexport the named export 'KnownArgumentNamesRule' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
57:0-59:675 Can't reexport the named export 'KnownDirectivesRule' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
57:0-59:675 Can't reexport the named export 'KnownFragmentNamesRule' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
57:0-59:675 Can't reexport the named export 'KnownTypeNamesRule' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
57:0-59:675 Can't reexport the named export 'LoneAnonymousOperationRule' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
57:0-59:675 Can't reexport the named export 'NoFragmentCyclesRule' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
57:0-59:675 Can't reexport the named export 'NoUndefinedVariablesRule' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
57:0-59:675 Can't reexport the named export 'NoUnusedFragmentsRule' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
57:0-59:675 Can't reexport the named export 'NoUnusedVariablesRule' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
57:0-59:675 Can't reexport the named export 'OverlappingFieldsCanBeMergedRule' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
57:0-59:675 Can't reexport the named export 'PossibleFragmentSpreadsRule' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
57:0-59:675 Can't reexport the named export 'ProvidedRequiredArgumentsRule' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
57:0-59:675 Can't reexport the named export 'ScalarLeafsRule' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
57:0-59:675 Can't reexport the named export 'SingleFieldSubscriptionsRule' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
57:0-59:675 Can't reexport the named export 'UniqueArgumentNamesRule' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
57:0-59:675 Can't reexport the named export 'UniqueDirectivesPerLocationRule' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
57:0-59:675 Can't reexport the named export 'UniqueFragmentNamesRule' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
57:0-59:675 Can't reexport the named export 'UniqueInputFieldNamesRule' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
57:0-59:675 Can't reexport the named export 'UniqueOperationNamesRule' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
57:0-59:675 Can't reexport the named export 'UniqueVariableNamesRule' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
57:0-59:675 Can't reexport the named export 'ValidationContext' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
57:0-59:675 Can't reexport the named export 'ValuesOfCorrectTypeRule' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
57:0-59:675 Can't reexport the named export 'VariablesAreInputTypesRule' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
57:0-59:675 Can't reexport the named export 'VariablesInAllowedPositionRule' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
57:0-59:675 Can't reexport the named export 'specifiedRules' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
57:0-59:675 Can't reexport the named export 'validate' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
61:0-64 Can't reexport the named export 'GraphQLError' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
61:0-64 Can't reexport the named export 'formatError' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
61:0-64 Can't reexport the named export 'printError' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
63:0-96:42 Can't reexport the named export 'BreakingChangeType' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
63:0-96:42 Can't reexport the named export 'DangerousChangeType' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
63:0-96:42 Can't reexport the named export 'TypeInfo' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
63:0-96:42 Can't reexport the named export 'assertValidName' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
63:0-96:42 Can't reexport the named export 'astFromValue' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
63:0-96:42 Can't reexport the named export 'buildASTSchema' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
63:0-96:42 Can't reexport the named export 'buildClientSchema' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
63:0-96:42 Can't reexport the named export 'buildSchema' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
63:0-96:42 Can't reexport the named export 'coerceValue' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
63:0-96:42 Can't reexport the named export 'concatAST' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
63:0-96:42 Can't reexport the named export 'doTypesOverlap' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
63:0-96:42 Can't reexport the named export 'extendSchema' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
63:0-96:42 Can't reexport the named export 'findBreakingChanges' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
63:0-96:42 Can't reexport the named export 'findDangerousChanges' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
63:0-96:42 Can't reexport the named export 'findDeprecatedUsages' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
63:0-96:42 Can't reexport the named export 'getDescription' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
63:0-96:42 Can't reexport the named export 'getIntrospectionQuery' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
63:0-96:42 Can't reexport the named export 'getOperationAST' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
63:0-96:42 Can't reexport the named export 'getOperationRootType' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
63:0-96:42 Can't reexport the named export 'introspectionFromSchema' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
63:0-96:42 Can't reexport the named export 'introspectionQuery' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
63:0-96:42 Can't reexport the named export 'isEqualType' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
63:0-96:42 Can't reexport the named export 'isTypeSubTypeOf' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
63:0-96:42 Can't reexport the named export 'isValidJSValue' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
63:0-96:42 Can't reexport the named export 'isValidLiteralValue' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
63:0-96:42 Can't reexport the named export 'isValidNameError' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
63:0-96:42 Can't reexport the named export 'lexicographicSortSchema' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
63:0-96:42 Can't reexport the named export 'printIntrospectionSchema' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
63:0-96:42 Can't reexport the named export 'printSchema' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
63:0-96:42 Can't reexport the named export 'printType' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
63:0-96:42 Can't reexport the named export 'separateOperations' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
63:0-96:42 Can't reexport the named export 'typeFromAST' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
63:0-96:42 Can't reexport the named export 'valueFromAST' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/graphql/index.mjs
63:0-96:42 Can't reexport the named export 'valueFromASTUntyped' from non EcmaScript module (only default export is available)
<pre>

**Comment by IvanGoncharov** [comment]
@Ben305 Can you reproduce the same issue with `0.13.2`?

**Comment by leovanhaaren** [comment]
I'm facing the same issue and it also happens with `0.13.2`

**Comment by Ben305** [comment]
@IvanGoncharov This snippet from `package.json` is working fine with Angular 6.0.3

```
"graphql": "git://github.com/graphql/graphql-js.git#npm"
```

**Comment by IvanGoncharov** [comment]
@Ben305 Looks like I found root cause: https://github.com/angular/angular-cli/issues/10744
So it's the issue with `angular-cli` and should be fixed in `v6.1.0-beta.1`.
Can you please try it?

**Comment by Ben305** [comment]
@IvanGoncharov Thanks for pointing me in the right direction.

Working with these `devDependencies`

```
"@angular-devkit/build-angular": "~0.6.8",
"@angular/cli": "^6.1.0-rc.0",
```
